packages/kbn-build/src/utils/scripts.js

```diff
@@ -1,4 +1,6 @@
+import { join } from 'path';
```

Comment:
`join` is unused. Is this code not linted?

Reply:
Ah, damn, looks like I forgot to add it to https://github.com/elastic/kibana/blob/master/src/dev/eslint/default_eslint_paths.js. Fixed now.